src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/QPack/QPackDecoder.cs

```diff
@@ -710,15 +710,15 @@ private void OnIndexedHeaderName(int index)
             _state = State.HeaderValueLength;
         }
 
-        private void OnIndexedHeaderNamePostBase(int index)
+        private static void OnIndexedHeaderNamePostBase(int index)
```

Comment:
I believe this file is mirrored with aspnetcore. It'd be good to validate whether it's going to break anything when that happens.

Reply:
I believe this should be fine as it's private and aspnetcore already had them static (they have CS1822 enabled) before sync with runtime removed it.

https://github.com/dotnet/aspnetcore/pull/39453